diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index a73c111..682904e 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -11,6 +11,9 @@ target_link_libraries(build_memory_index ${PROJECT_NAME})
 add_executable(overall_performance overall_performance.cpp)
 target_link_libraries(overall_performance ${PROJECT_NAME})
 
+add_executable(w1_canary w1_canary.cpp)
+target_link_libraries(w1_canary ${PROJECT_NAME})
+
 add_executable(test_insert_search test_insert_search.cpp)
 target_link_libraries(test_insert_search ${PROJECT_NAME})

